go.mod

```diff
@@ -18,6 +18,7 @@ require (
 	github.com/hashicorp/terraform-plugin-sdk/v2 v2.37.0
 	github.com/hashicorp/terraform-plugin-testing v1.12.0
 	github.com/jonboulle/clockwork v0.2.2
+	github.com/stretchr/testify v1.8.3
```

Comment:
We should minimize the number of dependencies unless strictly needed. In this case it's mostly helpers functions, so I believe we can write good tests with just the standard library.